### ISSUE TITLE
not ndpi capture interface with wireshark

### ISSUE THREAD

**Issue by sgmeire** [ntop/nDPI#507]
hi,
I've installed ndpi in wireshark like in "https://github.com/ntop/nDPI/tree/dev/wireshark" but i can't find a new extcap interface named "nDPI interface". Please, somebody can help me.
Thank you so much.

**Comment by kYroL01** [comment]
I think this part has some issue as reported in https://github.com/ntop/nDPI/issues/505

We have to check deeply

**Comment by tung-nguyen11** [comment]
i have the same issue.  i'm new to nDPI and ntopng, would someone walk me step by step how to get nDPI extcap interface show in wireshark?

**Comment by lucaderi** [comment]
Have you installed wireshark 2.6+ ?

**Comment by Lizeth2989** [comment]
Hey guys, 
I installed nDPI in wireshark using this guide https://sharkfesteurope.wireshark.org/assets/presentations17eu/19.pdf

First, I did it in Windows (wireshark 3.5) and the nDPI interface never appeared. Only Lua menu. 

Later, I used an ubuntu (wireshark 2.6) and the nDPI interface appeared. However, when I click on it, gives me this error 

"No packets capture, As no data was captured, closing the temporary capture file. Help ..."

Ofc I captured packets but I do not receive the wireshark interface options (ppt page 33). Someone can help me ! 

Thank you
